content/doc/book/installing/index.adoc

```diff
@@ -224,87 +491,59 @@ capabilities, rather than relying on the `SunOS` check, the above caveats for
 ZFS integration with Jenkins should be considered.
 
 
-=== macOS
-
-To install from the website, using a package:
-
-* link:http://mirrors.jenkins.io/osx/latest[Download the latest package]
-* Open the package and follow the instructions
-
-Jenkins can also be installed using `brew`:
-
-* Install the latest release version
-[source,bash]
-----
-brew install jenkins
-----
-
-* Install the LTS version
-[source,bash]
-----
-brew install jenkins-lts
-----
-
-
-=== Windows
-
-To install from the website, using the installer:
-
-* link:http://mirrors.jenkins.io/windows/latest[Download the latest package]
-* Open the package and follow the instructions
-
-
-=== Docker
-You must have link:http://docker.io[Docker] properly installed on your machine.
-See the link:https://www.docker.io/gettingstarted/#h_installation[Docker installation guide] for details.
-
-
-First, pull the official link:https://hub.docker.com/r/jenkins/jenkins[jenkins] image from Docker repository.
-
-[source,bash]
-----
-docker pull jenkins/jenkins
-----
-
-Next, run a container using this image and map data directory from the container to the host; e.g in the example below `/var/jenkins_home` from the container is mapped to `jenkins/` directory from the current path on the host. Jenkins `8080` port is also exposed to the host as `49001`.
-
-[source,bash]
-----
-docker run -d -p 49001:8080 -v $PWD/jenkins:/var/jenkins_home -t jenkins/jenkins
-----
-
-=== Other
-See link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
-
-
 [[setupwizard]]
-== Post-install (Setup Wizard)
+== Post-installation setup wizard
 
-=== Create Admin User and Password for Jenkins
+After downloading, installing and running Jenkins using one of the procedures
```

Comment:
Give an quick statement about what is about to happen.

Reply:
Done.